src/main/java/org/fundacion/jala/converter/core/parameter/VideoParameter.java

```diff
@@ -7,14 +7,11 @@
  *
```

Comment:
in this file should have a empty line between line 2 and 3

Reply:
Removed.